haiku-apps/beshare/beshare-2.32.recipe

```diff
@@ -36,27 +35,26 @@ REQUIRES="
 	lib:libsanta$secondaryArchSuffix
 	lib:libz
 	"
+
 BUILD_REQUIRES="
 	haiku${secondaryArchSuffix}_devel
 	devel:libsanta$secondaryArchSuffix
 	devel:libz$secondaryArchSuffix
-	#devel:muscle$secondaryArchSuffix
+#	devel:muscle$secondaryArchSuffix
 	"
 BUILD_PREREQUIRES="
 	cmd:gcc$secondaryArchSuffix
 	cmd:ld$secondaryArchSuffix
 	cmd:make
+	cmd:wget
 	makefile_engine
 	"
 
-PATCHES="beshare-$portVersion.patchset"
```

Comment:
I just noticed this line is being removed.
Well, if `haiku-apps/beshare/patches/beshare-2.31.patchset` is no longer needed for 2.32 then it might be a good idea to remove from the repo:
```
cd haikuports/haiku-apps/beshare/patches
git rm beshare-2.31.patchset
git commit --amend --no-edit beshare-2.31.patchset
```
I guess the patch is not needed anymore, but I would wait for a confirmation from  @pulkomandy and/or @mmuman.
BTW, if you have tried to build beshare-2.32, did it work fine?